modules/local/epytope_peptide_prediction.nf

```diff
@@ -37,6 +37,10 @@ process EPYTOPE_PEPTIDE_PREDICTION {
         argument = "--tool_thresholds ${params.tool_thresholds} " + argument
     }
 
+    if (params.affinity_thresholds) {
+        argument = "--affinity_thresholds " + argument
```

Comment:
This parameter should not need an argument (and see comment about parameter name above).

Reply:
Sorry, this is fine. 👍